metricbeat/magefile.go

```diff
@@ -133,7 +133,17 @@ func MockedTests(ctx context.Context) error {
 
 // Fields generates a fields.yml and fields.go for each module.
 func Fields() {
-	mg.Deps(fieldsYML, moduleFieldsGo)
+	mg.Deps(libbeatAndMetricbeatCommonFieldsGo, moduleFieldsGo)
+	mg.Deps(fieldsYML)
```

Comment:
Do we need the `fieldsYML` in addition. Shouldn't the module fields and common fields be enough as that is what is fed into the fieldsYML?